src/js/modules/modal-box.js

```diff
@@ -41,6 +41,11 @@ function closeModal() {
 		gratitude.classList.remove("open");
 	}
 
+	const focusableElements = activeModal.querySelectorAll("input, button");
```

Comment:
wtf man???
so much unnecessary code...

just use css to hide element from being in focus while it is closed

or there is an attribute "inert", but I still would prefer css

Reply:
Done

Reply:
please, use css for it

the less js, the better

Reply:
Done